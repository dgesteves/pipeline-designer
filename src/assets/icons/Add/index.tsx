import { TProps } from "../types";

export default function Add(props: TProps): JSX.Element {
  return (
    <svg
      xmlns="http://www.w3.org/2000/svg"
      width="1em"
      height="1em"
      viewBox="0 0 32 32"
      xmlSpace="preserve"
      {...props}
    >
      <path
        style={{
          fill: "currentColor",
        }}
        d="M16 4c6.6 0 12 5.4 12 12s-5.4 12-12 12S4 22.6 4 16 9.4 4 16 4m0-2C8.3 2 2 8.3 2 16s6.3 14 14 14 14-6.3 14-14S23.7 2 16 2z"
      />
      <path
        style={{
          fill: "currentColor",
        }}
        d="M24 15h-7V8h-2v7H8v2h7v7h2v-7h7z"
      />
      <path
        style={{
          fill: "none",
        }}
        d="M0 0h32v32H0z"
      />
    </svg>
  );
}
