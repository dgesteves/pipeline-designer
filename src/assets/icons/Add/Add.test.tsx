import { render } from "@testing-library/react";
import Add from "./index";

describe("Add icon", () => {
  it("should render correctly", () => {
    const { container } = render(<Add />);
    expect(container).toBeInTheDocument();
  });

  it("should have the correct svg", () => {
    const { container } = render(<Add />);
    expect(container).toMatchSnapshot();
  });
});
