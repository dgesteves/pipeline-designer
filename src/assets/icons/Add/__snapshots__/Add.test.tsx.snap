// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add icon should have the correct svg 1`] = `
<div>
  <svg
    height="1em"
    viewBox="0 0 32 32"
    width="1em"
    xml:space="preserve"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M16 4c6.6 0 12 5.4 12 12s-5.4 12-12 12S4 22.6 4 16 9.4 4 16 4m0-2C8.3 2 2 8.3 2 16s6.3 14 14 14 14-6.3 14-14S23.7 2 16 2z"
      style="fill: currentColor;"
    />
    <path
      d="M24 15h-7V8h-2v7H8v2h7v7h2v-7h7z"
      style="fill: currentColor;"
    />
    <path
      d="M0 0h32v32H0z"
      style="fill: none;"
    />
  </svg>
</div>
`;
